rocketpy/simulation/flight.py

```diff
@@ -3566,6 +3715,8 @@ def __init__(self, t, derivative=None, callbacks=None, clear=True):
                 self.derivative = derivative
                 self.callbacks = callbacks[:] if callbacks is not None else []
                 self.clear = clear
+                self.time_bound = None
+                self.TimeNodes = None
 
```

Comment:
I think we already changed this in the develop branch:

```suggestion
                self.time_nodes = None

```